src/components/input/TypeHandler/Base.ts

```diff
@@ -27,6 +28,9 @@ export class Base extends TypeHandler {
 	protected constructor(component: Component<any>) {
 		super(component)
 	}
+	handleBlur(state: State, event: FocusEvent): State {
```

Comment:
Why do we need `event`? Everything else in `TypeHandler ` does not require a browser to work. This is better for testing and reuse in other contexts.